src/utils/queue.rs

```diff
@@ -164,37 +149,4 @@ mod tests {
             Ok(())
         })
     }
-
-    #[test]
-    fn add_to_queue() {
-        wrapper(|env| {
-            let db = env.db();
-
-            let test_crates = [
-                ("rcc", "0.1.0", 2),
-                ("lasso", "0.1.0", -1),
-                ("hexponent", "0.1.0", 0),
-                ("destroy-all-humans", "0.0.0-alpha", -100000),
-                ("totally-not-destroying-humans", "0.0.1", 0),
-            ];
-
-            for (name, version, priority) in test_crates.iter() {
-                add_crate_to_queue(&db.conn(), name, version, *priority)?;
-
-                let query = db.conn().query(
-                    "SELECT name, version, priority FROM queue WHERE name = $1",
-                    &[&name],
-                )?;
-
-                assert!(query.len() == 1);
-                let row = query.iter().next().unwrap();
-
-                assert_eq!(&row.get::<_, String>(0), name);
-                assert_eq!(&row.get::<_, String>(1), version);
-                assert_eq!(row.get::<_, i32>(2), *priority);
-            }
-
-            Ok(())
-        })
-    }
```

Comment:
Where did this test go?

Reply:
It was split in the `test_add_and_process_crates` and `test_queued_crates`.